### DISCUSSION TITLE
Built-in leaderboards feedback

### DISCUSSION THREAD

**Discussion by AlexandreSi** [4ian/GDevelop#3827] [Enhancement ideas (and their technical discussions)]
Today 04/14/2022 has been released a beta version of GDevelop with built-in leaderboards.

Please use this discussion to bring us feedback as well as suggestions or bugs you may have encountered so that we can have a stable version that is ✨ **perfect** ✨!

**Comment by AlexandreSi** [comment]
Maybe add another action that would be:
- Save player score to leaderboard XXX (when running in preview, send to leaderboard YYY)

**Comment by jack-ii** [comment]
Good so far👍
A few ideas I have
+option for displaying a maximum of entries (e.g. for top10)
+option to show score formatted as time (00:00) and set the column title to "Time"
+option to not overlay the whole game
+option for background and text color (also transparent background would be good)
+add a button in the leaderboard-dashboard to clear the leaderboard (delete all entries at once)
+possibly a optional title text that can be set with an event action
![02](https://user-images.githubusercontent.com/81538128/163568214-1e36fe0a-058c-4417-b5d2-e79d909454c0.png)
![01](https://user-images.githubusercontent.com/81538128/163568216-c4345f1a-a5e0-43b7-a18c-31d7fb1eb4bb.png)

**Reply by AlexandreSi** [reply]
Thanks for the suggestions!

> +add a button in the leaderboard-dashboard to clear the leaderboard (delete all entries at once)

It's maybe not clear enough but you can use the "Reset" button to do so.

**Reply by jack-ii** [reply]
I think I don't see the button🤔

**Reply by AlexandreSi** [reply]
Check the wiki [here](https://wiki.gdevelop.io/gdevelop5/interface/games-dashboard/leaderboard-administration#reset_your_leaderboard)

**Reply by jack-ii** [reply]
ooh okay thx. that's really not clear enough🙃
maybe a reset button next to the delete button would be better.

**Comment by DADsPoker** [comment]
This is a great addition, I have currently added it and it is working great. THANK YOU. I am still testing. I will update if I find anything.

**Comment by AlexandreSi** [comment]
Seen from someone else: trim white spaces from player name before sending it

**Comment by AlexandreSi** [comment]
Seen from someone else: Allow user to select default leaderboard to be displayed on liluo

**Comment by Bouh** [comment]
~~There is no icon for the leaderboard category like other categories can have.~~ **4ian: fixed**
Might be worth adding a Yellow trophy.
![image](https://user-images.githubusercontent.com/1670670/164217678-36d0cca3-079f-4398-a5a8-259c7397a192.png)

**Reply by 4ian** [reply]
I'm fixing this

**Comment by Bouh** [comment]
When we play a game, the value is sent to the leaderboard, and we can see results in an integrated leaderboard in-game.
But on Liluo.io the leaderboard isn't refreshed when our new entry has been sent, I had to press F5 on the webpage, and lose where I was in the game.

**Comment by jack-ii** [comment]
Unregister a game does not work if the game previously had a leaderboard.
Testet in 5.0.132 final
![66](https://user-images.githubusercontent.com/81538128/165847025-5725cab6-386b-4c32-806e-763f79ea5176.png)

**Reply by Bouh** [reply]
You can still make the game undiscoverable on Liluo.

**Reply by AlexandreSi** [reply]
Oh I see what is the bug here, I'm on it.

**Reply by AlexandreSi** [reply]
Ok, this should be solved now. Thanks for reporting the bug!
